projects/Galene/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = "Galene is a self-hosted video conferencing server. It features advanced networking and video algorithms and automatic subtitling.";
+    subgrants = [
+      "Galene"
+    ];
+  };
+
+  nixos.modules.programs = {
+    galene = {
+      name = "galene";
+      module = ./module.nix;
```

The module already exists [in nixpkgs](https://github.com/NixOS/nixpkgs/blob/nixos-24.11/nixos/modules/services/web-apps/galene.nix)

```suggestion
      module = "${sources.inputs.nixpkgs}/nixos/modules/services/web-apps/galene.nix";
```